recipes/ri/bld.bat

```diff
@@ -0,0 +1,8 @@
+mv "%RECIPE_DIR%\sys" "%SRC_DIR%\sys" 
+sed -i '66s/.*/#include \"sys\/time.h\"/' include\timer.h
```

Comment:
you probably don't have sed on windows

Reply:
@scopatz - I have replaced the sed command in the .bat file with a patch (e617868).